contracts/libraries/StorageLib.sol

```diff
@@ -100,8 +100,8 @@ library StorageLib {
         assetToken_ = saveTokenStorage().insuranceToken;
     }
 
-    function uniswapFactory() internal view returns (address uniswapFactory_) {
-        uniswapFactory_ = saveTokenStorage().uniswapFactory;
+    function exchangeFactory() internal view returns (address uniswapFactory_) {
+        uniswapFactory_ = saveTokenStorage().exchangeFactory;
```

Comment:
I'll replace these remaining `uniswapFactory` references here and in `setAddresses` in my upcoming PR.